3696/3/app/controllers/place_controller.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require_relative '../helpers/info_helper'
+require_relative '../helpers/place_helper'
+require_relative '../helpers/auth_helper'
+
+class PlaceController < Sinatra::Base
+  set views: proc { File.join(root, '../views/') }
```

Comment:
as far as I know, you can define views directory without a proc
```
set :views, File.dirname(__FILE__)
```
or you use proc because of the `root`?

Reply:
Yep, because of the root. I think I will change it explicitly

Reply:
Hmm, to use your code I'll need to have main file in app directory(without any subdirectories). Doesn't look like a good idea